target-platforms/packages/dxp-7.4/package.json

```diff
@@ -170,5 +192,5 @@
 		"preversion": "yarn ci",
 		"test": "cd ../.. && yarn test"
 	},
-	"version": "5.0.0"
+	"version": "67.0.0"
```

Comment:
Why is our version our uXX version plus 1?

Reply:
See here -> https://github.com/liferay/liferay-frontend-projects/tree/master/target-platforms/packages/create-platform#liferay-dxp-tags (and save it as a bookmark because I have to visit it each time I have to modify or release target platforms :joy: ).

Basically :point_down: 

> The 7.4 structure is similar to the one in CE but using ga# for the first release of a major version series, and -u# for successive updates.

So, 1.x.x is for the initial release and 2.x.x for the second (which is `u1`).

In CE there's not that problem because the first release is `ga1`.